contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -776,12 +779,25 @@ library LiquidationLogic {
         ).collateralizedBalanceOf(params.borrower);
 
         // price of the asset that is used as collateral
-        if (INToken(superVars.collateralXToken).getAtomicPricingConfig()) {
+        if (
+            IXTokenType(superVars.collateralXToken).getXTokenType() ==
```

Comment:
in current design UniV3 is not atomic token because traitMultiplier is zero, better to rethink this and apply also balanceLimit on atomic portion